.gitignore

```diff
@@ -1,5 +1,6 @@
 *.gem
 *.rbc
+*.lock
```

Comment:
Gemfile.lock is [necessary to be committed](https://stackoverflow.com/questions/4151495/should-gemfile-lock-be-included-in-gitignore)

Reply:
I have removed this from .gitignore